docs/topics/proc_debug-start-v8-debug-nodejs-openshift.adoc

```diff
@@ -0,0 +1,75 @@
+= Starting your Application on OpenShift and Attaching the V8 Inspector
+The V8 inspector enables you to debug your {NodeJS}{ndash}based application using other tools, such as link:https://developers.google.com/web/tools/chrome-devtools/[Chrome DevTools], that use the link:https://chromedevtools.github.io/debugger-protocol-viewer/1-2/[Chrome Debugging Protocol].
+
+.Prerequisites 
+* An application you want to debug with `nodeshift` installed.
+* The `oc` CLI client installed.
+* `npm` installed.
+* The V8 inspector installed, such as the one provided in link:https://www.google.com/chrome/index.html[the Google Chrome Browser].
+
+
+.Procedure
+. Log in to your OpenShift instance with the `oc` client.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc login ...
+----
+
+. Create a new project in OpenShift.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc new-project {project-name}
+----
+
+. Use `nodeshift` to deploy the application to OpenShift.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm install && npm run openshift
```

Comment:
I wonder if we need to go into a little more depth on what "npm run openshift" is, or is that covered in another part of the docs.